SPECS/shim-unsigned-aarch64/shim-unsigned-aarch64.spec

```diff
@@ -1,83 +1,19 @@
 %global debug_package %{nil}
 Summary:        First stage UEFI bootloader
 Name:           shim-unsigned-aarch64
-Version:        15
-Release:        4%{?dist}
+Version:        15.3
+Release:        1%{?dist}
 URL:            https://github.com/rhboot/shim
 License:        BSD
 Vendor:         Microsoft
 Distribution:   Mariner
-Source0:        https://github.com/rhboot/shim/releases/download/15/shim-%{version}.tar.bz2
-Source1:        cbl-mariner-ca.der
+Source0:        https://github.com/rhboot/shim/releases/download/%{version}/shim-%{version}.tar.bz2
+Source1:        sbat.csv.in
+Source100:      cbl-mariner-ca-20210127.der
+BuildRequires:  dos2unix
+BuildRequires:  vim-extra
 ExclusiveArch:  aarch64
 
-BuildRequires:  gnu-efi
-BuildRequires:  gnu-efi-devel
```

Comment:
Double-checking: are these two packages no longer needed?

Reply:
Correct, those packages are no longer needed. Long story but shim-15.3 now git submodules a forked version of gnu-efi source code which contain some fixes. See https://github.com/rhboot/shim/commit/c61bfdc8a34fea1c6ea97ac0fb9c99f34f980381